### PR TITLE
BITMAKER-1840 estela: Configure repository descriptions

### PR DESCRIPTION
**Pull request by raymond1242**
We need to define the badges, links, description, and GitHub topics of estela repositories:
Consider all the github editable fields that offer a better understanding of the repositories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+
+[![version](https://img.shields.io/badge/version-0.1-blue)](https://github.com/bitmakerla/estela)
+[![python-version](https://img.shields.io/badge/python-v3.10-orange)](https://www.python.org)
+[![build](https://img.shields.io/badge/build-passing-brightgreen)](https://github.com/bitmakerla/estela/actions)
```
Comment (rodp63) [inline README.md]: Why is this pointing to the main estela repo?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+![CI](https://github.com/eslint/eslint/workflows/CI/badge.svg)
```
Comment (rodp63) [inline README.md]: This repo currently doesn’t have a CI

**Review by rodp63 — CHANGES_REQUESTED** [review]
* Please, put all the badges in one line.
* The 'about' section should describe this repository, check out this description: https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy

**Comment by raymond1242** [comment]
@rodp63 I think issues are resolved

**Review by rodp63 — APPROVED** [review]
Looks good

**Pull request merged**